.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# For more details on how to use this file, see
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file
+
+*       @jeffmay
```

Comment:
@russellremple can I add you here as well? Maybe I should add the whole @rallyhealth/scala-engineers team?